src/main/java/frc/robot/subsystems/ShooterPivot.java

```diff
@@ -97,12 +100,40 @@ private void setPosition(Rotation2d position) {
     setpoint = position.getRadians();
   }
 
+  public Rotation2d getEstimatedAngle(double distance) {
+    double angle = Constants.ShooterPivot.a * distance + Constants.ShooterPivot.b;
+    return Rotation2d.fromDegrees(angle);
+  }
+
+  public double speakerDistance(Pose2d RobotPose) {
```

Comment:
Rephrase to getSpeakerDistance for clarity, RobotPose should be lower camel case (robotPose)